package.json

```diff
@@ -15,22 +15,24 @@
   },
   "main": "lib/kv-redis.js",
   "scripts": {
-    "test": "mocha test/integration/*.js",
+    "test": "mocha test/unit/*.js && mocha test/integration/*.js",
```

Comment:
I believe we should get the same effect by running `mocha test/unit/*.js test/integration.js`. However, `--bail` wouldn't have the same effect as it would only show a single error (the first one it fails on -- the current behaviour is it shows **all** failing unit tests or all integration tests together). That said, I'm OK with `--bail` too as we should be fixing all failing tests anyways so whether we display them altogether at once or one by one via bail doesn't matter to me.